dokumentlager/domene/src/main/java/no/nav/foreldrepenger/melding/dokumentdata/DokumentTypeId.java

```diff
@@ -9,7 +9,6 @@
  */
 public class DokumentTypeId {
 
-    public static final DokumentTypeId SØKNAD_FORELDREPENGER_FØDSEL = new DokumentTypeId("I000005"); //$NON-NLS-1$
```

Comment:
Er det noe vi skal slette?

Reply:
Jeg slettet den fordi den ikke er i bruk. Hvis vi får behov for den kan den opprettes igjen :-)